tests/lib/rules/no-restricted-imports.js

```diff
@@ -35,6 +35,134 @@ ruleTester.run("no-restricted-imports", rule, {
         {
             code: "import withGitignores from \"foo/bar\";",
             options: [{ patterns: ["foo/*", "!foo/bar"] }]
+        },
+        {
+            code: "import AllowedObject from \"foo\";",
+            options: [{
+                paths: [{
+                    name: "foo",
+                    importNames: ["DisallowedObject"]
+                }]
+            }]
+        },
+        {
+            code: "import DisallowedObject from \"foo\";",
+            options: [{
+                paths: [{
+                    name: "foo",
+                    importNames: ["DisallowedObject"]
+                }]
+            }]
+        },
+        {
+            code: "import * as DisallowedObject from \"foo\";",
+            options: [{
+                paths: [{
+                    name: "bar",
+                    importNames: ["DisallowedObject"],
```

Comment:
So it’s fine if i call my ModuleRecord from this path anything else, just not this name?